src/Model/EventModel.php

```diff
@@ -39,11 +41,16 @@ public function __construct(array $config ) {
             $config['customer_properties']
         );
         $this->properties = $config['properties'];
-        // Can pass in unix timestamp if prefixed with '@'. Else just let it parse the date
```

Comment:
Can you update the docstring for this param to mention that it accepts either a unix timestamp OR any date format that DateTime accepts?